models/news.models.js

```diff
@@ -25,3 +25,16 @@ exports.selectArticleCommentsById = (article_id) => {
 		return comments.rows
 	})
 }
+
+exports.updateArticleVotes = (article_id, body) => {
+	const values = [article_id, body]
+	const pSelectArticle = this.selectArticle(article_id)
+	const pUpdate = db.query('UPDATE articles SET votes = votes + $2 WHERE article_id = $1 RETURNING *;', values)
+	return Promise.all([pSelectArticle, pUpdate])
+	.then( ([article, update]) => {
```

Comment:
I think article is an unused variable here - ideally you want to avoid them as its taking up memory, then not being used. 
if you switched the order of the promises, does it still work?
You could then just have the one argument i.e. update
naming wise `updatedArticle` would be better